### PR TITLE
Adds tab ui component in bt-classic-api test

### PR DESCRIPTION
**Pull request by cathook**
Added files:
  `js/classic/utils.js` -- Contains some utility functions for classic-api test.
  `js/classic/ui.js` -- Contains all ui components for classic-api test.
  `js/classic/main.js` -- Main entries point of classic-api test.
  `style/classic/ui.css` -- CSS for ui components of classic-api test.

Added dependency libraries: `jquery`, `jquery-ui`

### CODE REVIEW

**Comment by cathook** [comment]
@dwi2 please review it, thanks.

File: js/classic/main.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+(function(exports) {
+  'use stricts';
+
+  exports.classic = exports.classic || {};
+
+  var classic = exports.classic;
+  var ui = exports.classic.ui;
+
+
```
Comment (dwi2) [inline js/classic/main.js]: nit: Please remove this line. One blank line for indention is enough.

File: js/classic/utils.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  /**
+   * Returns an unique string each time it is called.
+   *
+   * @param prefix (optional) the prefix of the returned string.
+   * @return A string which will never equal to strings returned in other time.
+   */
+  utils.createUniqueId = (function() {
+    var counter = {};
+    return function(prefix) {
+      if (prefix == undefined) {
```
Comment (dwi2) [inline js/classic/utils.js]: We should avoid vague comparison. I think we should change this line either to

``` js
if (!prefix) {
```

or

``` js
if (prefix === undefined) {
```

File: js/classic/main.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+          tabsManager.removeTab(this._me._tab);
+          this._me._tab.destroy();
+        }
+      };
+      this._tab = new ui.Tab(this._eventHandler);
+      this._tab.name = 'tab ' + id;
+      tabsManager.addTab(this._tab);
+    };
+    for (var i = 0; i < 4; ++i) {
+      new FakeBackend(i);
```
Comment (dwi2) [inline js/classic/main.js:26]: I am just wondering: Don't we need to hold these instance of `FakeBackend` somewhere?

**Comment by dwi2** [comment]
:+1: It looks good to me in general. Just some few nits. Please ping me again after you fix them. 
Thanks.
[LGTM](http://lgtm.in/i/yHbgBNSgn)

File: js/classic/main.js (hunk 3 above)
Comment (cathook) [inline js/classic/main.js:26]: This is just for test.  And it works either if I don't hold the instance of `FakeBackend`.

**Pull request merged**